design/use_cases/_template_use_case.md

```diff
@@ -0,0 +1,16 @@
+# Use case Title
+As a <Developer, Researcher, Technician, etc.> I want the robot to <action> so that <I, the robot, etc.> can <do something important>
+
+## More details
+- Why is this needed?
+- What is the expected user interaction? 
+
```

Comment:
I suggest we add: "Are there any non-functional requirements associated with this use case?"